WPILib.Extras/CommandRobot.cs

```diff
@@ -0,0 +1,26 @@
+﻿using WPILib.Commands;
+
+namespace WPILib.Extras
+{
+    /// <summary>
+    /// Implements the boilerplate needed to use an <see cref="IterativeRobot"/>
+    /// with the command-based model, by running the scheduler as needed.
+    /// </summary>
+    public abstract class CommandRobot : IterativeRobot
+    {
+        // This function is called periodically while disabled
+        public override void DisabledPeriodic() => Scheduler.Instance.Run();
```

Comment:
Oh yeah I forgot. I thought override sealed, but its the other way around. So this part is good.